docs/reference/search/search-your-data/search-template.asciidoc

```diff
@@ -17,11 +17,14 @@ your searches without modifying your app's code.
 To create or update a search template, use the <<create-stored-script-api,create
 stored script API>>.
 
+
 The request's `source` supports the same parameters as the
 <<search-search-api-request-body,search API>>'s request body. `source` also
-supports https://mustache.github.io/[Mustache] variables, typically enclosed in
+supports https://mustache.github.io/[Mustache] variables and has a https://github.com/elastic/elasticsearch/blob/main/modules/lang-mustache/build.gradle/[dependency] on the open source project https://github.com/spullara/mustache.java[mustache.java]. 
```

Comment:
Why is it relevant for our documentation that we have a gradle dependency on Mustache? Can we either omit that fact or elaborate on it?

Reply:
I think the main intention was to let user know the dependency. And assume that user could read more about Mustache in their GitHub repo